server/src/main/java/org/elasticsearch/action/admin/cluster/storedscripts/PutStoredScriptRequest.java

```diff
@@ -166,4 +168,16 @@ public String toString() {
             (context != null ? ", context [" + context + "]" : "") +
             ", content [" + source + "]}";
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
```

Comment:
given that you added this method, I would also add a unit test for it. I would probably also move the respective parsing code from RestPutStoredScriptAction here as a fromXContent method so that testing is also easier.

Reply:
can you add a unit test for this method please?

Reply:
I was trying to find a better way to add a unit test for this method. After reading some classes I found that those classes which have `toXContent` method also have `assertToXContentBody` assertion. so I add something like that in `RequestConvertersTests.java`

https://github.com/johnny94/elasticsearch/blob/82a8611fa631d4b4d9d0b25e011a2900c5674443/client/rest-high-level/src/test/java/org/elasticsearch/client/RequestConvertersTests.java#L2092

Is this a proper way to test the code?

Reply:
that is part of it, but it only tests that the client takes the request body into account and prints it out. It would also be nice to add a test to the server-side code, which verifies the output itself. We usually do this by adding a request test. You can just add a new testToXContent method to `PutStoredScriptRequestTests` which tests the xcontent output. Something static (not randomized) is fine for this type of test.

Reply:
Thanks for your hint! I have added a test for the method.